scripts/http-wordpress-contentinjection.nse

```diff
@@ -0,0 +1,107 @@
+local http = require "http"
+local shortport = require "shortport"
+local string = require "string"
+local stdnse = require "stdnse"
+local url = require "url"
+local vulns = require "vulns"
+local json = require "json"
+
+description = [[ 
+There is privilege escalation vulnerability in Wordpress
+  Rest API. WordPress Versions 4.7.0 and 4.7.1 are known to be affected. The
+  vulnerability allows the visitor to edit any post and replace it with whatever
+  the attacker wants.
+
+  This is a serious vulnerability that can be misused in different ways to 
+  compromise a vulnerable site. 
```

Comment:
This last paragraph is copy-and-pasted from the Sucuri website and is not really needed here. Best to leave it out.